src/screens/SendReceive/Send/AssetsScreen/AssetRow.tsx

```diff
@@ -64,7 +67,19 @@ const AssetRow = ({ asset, style, isLast }: AssetRowProps) => {
 
     const cleanedAmount = isNumericStringValid(inputAmount, true) ? inputAmount : ''
 
-    setAmount(cleanedAmount)
+    setAmount((prevAmount) => {
+      const amountLength = cleanedAmount.length
+
+      if (amountLength >= 10) {
+        if (amountLength < 20) {
+          setInputFontSize((prevSize) => prevSize + (prevAmount.length - amountLength))
+        }
+      } else {
+        setInputFontSize(23)
+      }
```

Comment:
nit: what about this?

```suggestion
      setInputFontSize((prevSize) =>
        amountLength < 10 ? 23 : amountLength < 20 ? prevSize + (prevAmount.length - amountLength) : prevSize
      )
```

Reply:
We need to use this logic in every place we use `setAmount` with a user defined value. This is the case when using `handleUseMaxAmountPress` as well.

I'll take care of both of the above in a commit, if you don't mind.

Reply:
When going back and forth between the send flow screens after the amounts have been entered, the font size remains the default one (23)